docs/developers/user-impersonation.mdx

```diff
@@ -140,6 +140,12 @@ grant_type=urn:ietf:params:oauth:grant-type:token-exchange
 
 The `access_token` returned will be bound to the specified resource, ensuring it can only be used with TechCorp's customer data API.
 
+**Note**: For traditional web applications, include `client credentials` in the header of the token request to prevent a 401 invalid_client error.
+
+```json
+Authorization: Basic ${Buffer.from(${id}:${secret}, 'utf8').toString('base64')}
+```
```

Comment:
This is inaccurate I think.
1. First of all, not all "traditional web" are JS based framework, the code example showed here is Node.js
2. Even in Node.js, the code sample is incorrect. Should be
```
Authorization: Basic `${Buffer.from(`${id}:${secret}`, 'utf8').toString('base64')}`
```